app/controllers/api/groups_controller.rb

```diff
@@ -13,4 +14,17 @@ def subgroups
     respond_with_collection
   end
 
+  def upload_photo
```

Comment:
The code for updating cover / logo is identical, so I really don't want to have two actions for it. Since it's a multipart POST request, we can't include params and a file at the same time without hoop jumping, thus this `group/:key/upload_photo/:kind` route